metacat-connector-snowflake/src/main/java/com/netflix/metacat/connector/snowflake/SnowflakeConnectorTableService.java

```diff
@@ -152,12 +152,16 @@ protected ResultSet getColumns(
         @Nonnull @NonNull final Connection connection,
         @Nonnull @NonNull final QualifiedName name
     ) throws SQLException {
```

Comment:
This wouldn't be true anymore right? Looks like the caller of this method, get() in JdbcConnectorTableService already catches this. Wonder if this fix to go through the ExceptionMapper should be done there instead to catch similar exceptions from other methods?

Reply:
True. Your suggestion is right but that was the case before we made a bug fix.